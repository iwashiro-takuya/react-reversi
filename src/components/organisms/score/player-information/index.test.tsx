import React from 'react';
import renderer from 'react-test-renderer';
import Const from 'src/const';
import PlayerInformation from './index';

const { Player } = Const;

describe('Square unit test', () => {
  it('snapshot(Player White)', () => {
    const component = renderer.create(
      <PlayerInformation player={Player.WHITE} />,
    );
    expect(component).toMatchSnapshot();
  });
  it('snapshot(Player Black)', () => {
    const component = renderer.create(
      <PlayerInformation player={Player.BLACK} />,
    );
    expect(component).toMatchSnapshot();
  });
});
