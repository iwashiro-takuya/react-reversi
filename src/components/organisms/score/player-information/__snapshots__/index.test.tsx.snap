// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Square unit test snapshot(Player Black) 1`] = `
<div
  className="sc-gtssRu hAvyXn"
>
  <span
    className="sc-hKFyIo izrEJU"
  >
    2
  </span>
  <div
    className="sc-dlnjPT jUuLpY"
  >
    <span
      className="sc-bdnylx kjkACR"
      color="#000"
    />
  </div>
</div>
`;

exports[`Square unit test snapshot(Player White) 1`] = `
<div
  className="sc-gtssRu hAvyXn"
>
  <div
    className="sc-dlnjPT jUuLpY"
  >
    <span
      className="sc-bdnylx hEHfkv"
      color="#fff"
    />
  </div>
  <span
    className="sc-hKFyIo izrEJU"
  >
    2
  </span>
</div>
`;
